includes/stdlib.c

```diff
@@ -42,3 +42,10 @@ void* bsearch(const void *key, const void *ptr, size_t count, size_t size, int (
 
   return NULL;
 }
+
+int pthread_once(pthread_once_t *once_control,void (*init_routine)(void)) {
+  // Not actually once, just call it
+  int top;
+  init_routine();
+  return top;
+}
```

Comment:
I think we should keep these stub implementations organized according to their headers, so this should maybe go into a new file `pthread.c`.